apps/storybook-react/.storybook/preview.tsx

```diff
@@ -38,26 +38,26 @@ function withColorScheme(Story, context) {
 
   if (colorScheme === 'light') {
     return (
-      <Wrapper data-theme="light">
+      <Wrapper className="light">
```

Comment:
tesing classname instead of data-theme, data-theme still works but we are updating to classname to align with tailwind